plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2577,15 +2577,23 @@ producer_plugin_impl::push_result producer_plugin_impl::push_transaction(const f
    if (!disable_subjective_enforcement)
       sub_bill = subjective_bill.get_subjective_bill(first_auth, fc::time_point::now());
 
-   const auto prev_elapsed_time_us = trx->elapsed_time_us;
+   auto prev_elapsed_time_us = trx->elapsed_time_us;
    const auto prev_billed_cpu_time_us = trx->billed_cpu_time_us;
    if (in_producing_mode() && prev_elapsed_time_us > 0) {
+      const fc::microseconds block_time_remaining_us = block_deadline - start;
```

Comment:
nit: I'd move this line closer to where it is used (before #2594)

Reply:
https://github.com/AntelopeIO/spring/pull/1803/commits/c817915663647803e9d1b7c24032b4b5021ddf6f